### PR TITLE
Upgrade woodstox core asl version

### PR DESCRIPTION
**Pull request by Vitalij-Voronkoff**
Requires:BroadleafCommerce/BroadleafCommerce#2089

This upgrade needs to fix conflict old libraries needed to Apache CXF. BroadleafCommerce/blc-cybersource-payment#15

### CODE REVIEW

File: common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java [hunk 1]
```diff
@@ -112,45 +112,9 @@ public void testRequestMapProperty() {
...
-     * If a rule is executed calling the intersection method and a null is passed in for the first parameter, mvel will
-     * not know which method to pick, so it will end up picking one of them. This has proven to be undetermined at runtime,
-     * which has made the problem even harder to identify. Furthermore, once the choice is made during compilation,
-     * that compiled expression will utilize the "incorrect" choice going forward, which will cause the expression to
-     * fail, even when neither param is null. In fact, in the case above, mvel will "coerce" the first param Iterable
-     * instance into a String (i.e. flatten the list representation into a comma delimited String) in order to continue
-     * calling the method identified during compilation. This causes the expression to permanently be in an incorrect
-     * state from which it will never recover.
-     */
-    public void testMvelMethodOverloadFailureCase() throws IOException {
```
Comment (phillipuniverse) [inline common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java:136]: Why did you delete this test?

File: pom.xml [hunk 2]
```diff
@@ -1114,7 +1114,7 @@
             <dependency>
                 <groupId>org.mvel</groupId>
                 <artifactId>mvel2</artifactId>
-                <version>2.2.1.Final</version>
+                <version>2.4.3.Final</version>
```
Comment (phillipuniverse) [inline pom.xml:1117]: Why did you need to update mvel?

**Review by phillipuniverse — CHANGES_REQUESTED** [review]
I don't really have the context to approve/deny this PR, but the MVEL change seems suspect.

File: common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java (hunk 1 above)
Comment (Vitalij-Voronkoff) [reply common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java:136]: this test failures while project building. I found that in 5.2 this test was deleted as not necessary because mvel version upgraded. I need to build framework version 5.0 because I need to upgrade woodstox library version, because old version causes error working with Apache CXF

Comment (Vitalij-Voronkoff) [reply common/src/test/java/org/broadleafcommerce/common/rule/MvelHelperTest.java:136]: Context this task https://github.com/BroadleafCommerce/blc-cybersource-payment/pull/15

**Review by riteshadhikari17 — APPROVED** [review]

**Comment by phillipuniverse** [comment]
@riteshadhikari17 it looks like the questions that I had and the reason that I requested changes went unanswered. What is the reason for us upgrading this version of MVEL in this 5.0 version?

**Comment by danielcolgrove** [comment]
@Vitalij-Voronkoff has been on vacation.  I suspect it's a dependency caused by the upgrade from Axis1 to Axis2.

**Comment by riteshadhikari17** [comment]
@phillipuniverse Its actually upgrade from axis1 to cxf. Vitali has notes on this PR https://github.com/BroadleafCommerce/blc-cybersource-payment/pull/15

**Comment by danielcolgrove** [comment]
Yes, correct....not axis2.

**Comment by phillipuniverse** [comment]
@danielcolgrove @riteshadhikari17 I think you guys are missing the point, or maybe I am? This PR includes an upgrade to MVEL. Why does this PR have anything to do with MVEL? What does MVEL have to do at all with axis or cxf?

If the issue was a test failure _without_ upgrading the MVEL version, then it sounds like that test failure was legitimate.

**Comment by danielcolgrove** [comment]
@phillipuniverse I understand the question.  I would need to defer to Vitalii as to why it was required.  Upgrading MVEL wasn't part of the original unit of work.  I suspect the upgrade came in as a series of dependency upgrades.  Vitalii is back toward the end of the week and we can get more details once he returns.  @riteshadhikari17 If it makes sense from a timing perspective to revert this issue, let's do that.
